statsmodels/genmod/families/family.py

```diff
@@ -516,16 +516,16 @@ def loglike(self, endog, mu, scale=1.):
         llf = sum((`endog`*`mu`-`mu`**2/2)/`scale` - `endog`**2/(2*`scale`) - \
             (1/2.)*log(2*pi*`scale`))
         """
-        if isinstance(self.link, L.Power) and self.link.power == 1:
-            # This is just the loglikelihood for classical OLS
-            nobs2 = endog.shape[0]/2.
-            SSR = np.sum((endog-self.fitted(mu))**2, axis=0)
-            llf = -np.log(SSR) * nobs2
-            llf -= (1+np.log(np.pi/nobs2))*nobs2
-            return llf
-        else:
+        # if isinstance(self.link, L.Power) and self.link.power == 1:
```

Comment:
Kerby has prepared PR #2353 
I wanted to postpone it for 0.8 because I'm not sure about the API and impact.
I see if I can figure out #2353 in the next few days with examples and unit tests. #2353 will be merged before MixedLM, so this can always be adjusted later.